addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -741,6 +743,12 @@ var SnippetEditor = Widget.extend({
             if (val['drop-in']) {
                 this.selectorChildren.push(val['drop-in']);
             }
+            if (val['drop-lock-within']) {
+                this.selectorLockWithin.add(val['drop-lock-within']);
+            }
+            if (val['drop-exclude-ancestor']) {
+                selectorExcludeParent.add(val['drop-exclude-ancestor']);
```

Comment:
I really think it would be clearer to not use the parent word.. You should keep using ancestor. Can you make the change everywhere in the code and in the comments.